package.json

```diff
@@ -33,8 +33,8 @@
   "homepage": "https://github.com/revdotcom/revai-node-sdk#readme",
   "dependencies": {
     "axios": "^0.18.0",
-    "express": "^4.16.4",
-    "form-data": "^2.3.3"
+    "form-data": "^2.3.3",
```

Comment:
make sure to run npm install locally and add the changes to package-lock.json to match any updates made to dependencies

Reply:
I've been using npm uninstall to get rid of packages so package-lock is up to date. :)